generator/templates/proposal.template.ts

```diff
@@ -23,10 +27,11 @@ export const proposalTemplate = (
   * @title ${title || 'TODO'}
   * @author ${author || 'TODO'}
   * - discussion: ${discussion || 'TODO'}
-  * - deploy-command: make run-script contract=src/contracts/updates/${folderName}/${contractName}.sol:${contractName} network=${getChainAlias(chain)} broadcast=false generate_diff=true
+  * - deploy-command: make run-script contract=src/contracts/updates/${folderName}/${contractName}.sol:${contractName} network=${getChainAlias(
+    chain
+  )} broadcast=false generate_diff=true
   */
- contract ${contractName} is ${`RiskStewards${chain}`
- } {
+ contract ${contractName} is ${`RiskStewards${riskStewardChain}`} {
```

Comment:
```suggestion
 contract ${contractName} is ${`RiskStewards${chain === 'Base' ? 'BaseChain' : chain}`} {
```
I would put the condition here directly instead of declaring a new constant

Reply:
Pushed a fix (cause with this suggestion I needed to also remove the riskStewardChain const